packages/llama/src/token-addresses.ts

```diff
@@ -0,0 +1,7 @@
+import { type Address } from 'viem'
+import { OPTIMISM_CHAIN_ID } from './chain-ids.js'
+
+export const LLAMA_TOKENS: Record<number, Address[]> = {
+  // TODO arguably we should support any token here?  it's a governance token right that's arbitraryq
```

Comment:
Yeah we need an `other` component and better interface validation. For now best to hardcode the addresses.